public/app/features/dashboard-scene/settings/variables/utils.test.ts

```diff
@@ -27,12 +30,44 @@ import {
   hasVariableOptions,
   EditableVariableType,
   getDefinition,
+  getOptionDataSourceTypes,
 } from './utils';
 
 const templateSrv = {
   getAdhocFilters: jest.fn().mockReturnValue([{ key: 'origKey', operator: '=', value: '' }]),
 } as unknown as TemplateSrv;
 
+const dsMock: DataSourceApi = {
+  meta: {
+    id: DASHBOARD_DATASOURCE_PLUGIN_ID,
+  },
+  name: SHARED_DASHBOARD_QUERY,
+  type: SHARED_DASHBOARD_QUERY,
+  uid: SHARED_DASHBOARD_QUERY,
+  getRef: () => {
+    return { type: SHARED_DASHBOARD_QUERY, uid: SHARED_DASHBOARD_QUERY };
+  },
+} as DataSourceApi<DataQuery, DataSourceJsonData, {}>;
+
+jest.mock('@grafana/runtime', () => ({
+  ...jest.requireActual('@grafana/runtime'),
+  getDataSourceSrv: () => ({
+    get: async () => dsMock,
```

Comment:
@dprokop this did the trick to fix the unit test 😄

Reply:
@ivanortegaalba, This is relevant for your Query Editor Migration PR, the tests are failing for the same reason 😄 https://drone.grafana.net/grafana/grafana/155958/1/6